packages/iTwinUI-react/src/core/Wizard/Wizard.test.tsx

```diff
@@ -11,240 +11,240 @@ import {
+  const wizard = (
+    <Wizard
+      currentStep={2}
+      type='long'
+      steps={[
+        {
+          name: 'Step One',
+        },
+        {
+          name: 'Step Two',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  const { getByText, queryByText } = render(wizard);
+
+  expect(getByText('Step 3 of 3:')).toHaveClass(
+    'iui-stepper-steps-label-count',
+  );
+  expect(getByText('Step Three')).toHaveClass('iui-stepper-steps-label');
+
+  getByText('1');
+  getByText('2');
+  getByText('3');
+
+  expect(queryByText('Step One')).toBeNull();
+  expect(queryByText('Step Two')).toBeNull();
+});
+
+it('should display localized string in long wizard', () => {
+  const wizard = (
+    <Wizard
+      currentStep={1}
+      type='long'
+      localization={{
+        stepsCountLabel: (currentStep, totalSteps) =>
+          `This is ${currentStep} of great ${totalSteps}:`,
+      }}
+      steps={[
+        {
+          name: 'Step One',
+        },
+        {
+          name: 'Step Two',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  const { getByText } = render(wizard);
+
+  getByText('This is 2 of great 3:');
+  getByText('Step Two');
+});
+
+it('should display all step names but no numbers in workflow diagram', () => {
+  const wizard = (
+    <Wizard
+      currentStep={2}
+      type='workflow'
+      steps={[
+        {
+          name: 'Step One',
+        },
+        {
+          name: 'Step Two',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  const { getByText, queryByText } = render(wizard);
+
+  getByText('Step One');
+  getByText('Step Two');
+  getByText('Step Three');
+
+  expect(queryByText('1')).toBeNull();
+  expect(queryByText('2')).toBeNull();
+  expect(queryByText('3')).toBeNull();
+});
+
+it('should display tooltip upon hovering step if description provided', async () => {
+  jest.useFakeTimers();
+
+  const wizard = (
+    <Wizard
+      currentStep={1}
+      steps={[
+        {
+          name: 'Step One',
+          description: 'Step one tooltip',
+        },
+        {
+          name: 'Step Two',
+          description: 'Step two tooltip',
+        },
+        {
+          name: 'Step Three',
+        },
+      ]}
+    />
+  );
+
+  render(wizard);
+
+  expect(document.querySelector('.iui-tooltip')).toBeNull();
+  expect(screen.queryByText('Step one tooltip')).toBeNull();
+  fireEvent.mouseEnter(screen.getByText('Step One'), { bubbles: true });
+  const tooltip = document.querySelector('.iui-tooltip') as HTMLElement;
+  expect(tooltip).toBeVisible();
+  expect(tooltip).toHaveTextContent('Step one tooltip');
+
+  fireEvent.mouseLeave(screen.getByText('Step One'), { bubbles: true });
+  await waitForElementToBeRemoved(tooltip);
+
+  fireEvent.mouseEnter(screen.getByText('Step Three'), { bubbles: true });
+  expect(screen.queryByText('Step three tooltip')).toBeNull();
```

Comment:
Same thing I said in WorkflowDiagram.test.tsx

Reply:
Test changed